railties/lib/rails/generators/rails/app/templates/bin/dev.tt

```diff
@@ -1 +1,34 @@
-exec "./bin/rails", "server", *ARGV
+ENV["PORT"] ||= "3000"
+VERBOSE = ARGV.include?("--verbose") || ARGV.include?("-v") || ENV["VERBOSE"]
+RAILS_ARGS = ARGV - [ "--verbose", "-v" ]
```

Comment:
I am thinking of splitting the args with `--` instead.
`bin/dev -v -- -b 0.0.0.0`
This way if we want to add more options to `bin/dev`, filtering them out won't get too messy.